codalab/lib/beam/filesystems.py

```diff
@@ -0,0 +1,24 @@
+import os
+from apache_beam.io.filesystems import FileSystems
+from .blobstoragefilesystem import BlobStorageFileSystem
+from azure.storage.blob import BlobServiceClient
+from azure.core.exceptions import ResourceExistsError
+
+"""
+Modifies Beam to add support for Azure Blob Storage filesystems.
+
+Based on work from https://github.com/apache/beam/pull/12581. Once that PR is added into Beam,
```

Comment:
@percyliang see this regarding where those two files (the copied verbatim files -- blobstoragefilesystem.py and blobstorageio.py) came from. Although that PR has been merged into beam, a new version hasn't yet been released with those changes, so we may just have to keep those two files in for now.